srcpkgs/xfce4-calculator-plugin/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'xfce4-calculator-plugin'
+pkgname=xfce4-calculator-plugin
+version=0.7.1
+revision=1
+build_style=gnu-configure
+configure_args="--with-locales-dir=/usr/share/locale"
+hostmakedepends="pkg-config intltool"
+makedepends="xfce4-panel-devel"
+short_desc="Battery monitor plugin for the Xfce panel"
```

Comment:
The description doesn't look like it matches the name.

Reply:
> The description doesn't look like it matches the name.

Fixed

Reply:
ha